specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -841,6 +893,156 @@
           "x-ms-secret": true
         }
       }
+    },
+    "WorkloadProfile": {
+      "description": "Description of a workload profile.",
+      "type": "object",
+      "properties": {
+        "workloadProfileType": {
+          "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileType",
+          "description": "Workload profile type for the workloads to run on."
+        },
+        "provisioningState": {
+          "description": "Provisioning state of the workload profile",
+          "type": "string",
+          "readOnly": true
+        },
+        "minimum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum capacity."
+        },
+        "maximum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The maximum capacity."
+        }
+      },
+      "required": [
+        "workloadProfileType"
+      ]
+    },
+    "Certificate": {
```

Comment:
>Certificate [](http://example.com/codeflow?start=5&length=11)

Isn't this already defined in the commondefinitions?

Reply:
Fixed rebase issue.